app/routes/user.js

```diff
@@ -41,4 +42,19 @@ module.exports = function(router) {
             response.redirect('/login');
         }
     });
+
+    router.get('/forgotMyPassword', (request, response) => {
```

Comment:
Personal preference (and super nit-picky), but I think camel cased URLs look a bit unusual on the web. I prefer `/forgot-password` and `/reset-password/:resetToken`, but if you prefer this style it's fine. Not like people are going to be typing this in.